org-journal.el

```diff
@@ -2,7 +2,7 @@
 
 ;; Author: Bastian Bechtold
```

Comment:
Please add yourself to the Author list, @cslux! Github tells me that your contributions are bigger than my own!